homework_week_14/src/main/java/homework/Main.java

```diff
@@ -0,0 +1,168 @@
+package homework;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Main {
+    public static void main(String[] args) {
+
+        /*
```

Comment:
before merging please extract the functionalities in different methods (even in other classes) and compare how the code structure looks and if it makes more sense as a standard to separate concerns